### PR TITLE
docs(natvie-select): add a11y docs

### LINKED ISSUE #1568: Add accessibility section for `native-select`
### Subject

accessibility section for `native-select`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )
https://mantine.dev/core/native-select/#accessibility

#### Include the following content
* aria, tabindex property is supported

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by mei28**
Closes #1568 

## Description

add `native-select` docs for accessibility

## Current behavior (updates)

in English.
image

in Japanese
image

### CODE REVIEW

File: docs/contents/components/forms/native-select/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| `Space`, `Enter` | フォーカスしているオプションをアクティブにします。                                                         | -    |
+| `Home`           | 無効ではない最初のオプションをフォーカスしてアクティブにします。                                           | -    |
+| `End`            | 無効ではない最後のオプションをフォーカスしてアクティブにします。                                           |
+
+## ARIAロールと属性
+
+| コンポーネント           | ロールと属性         | 使い方                                                   |
+| ------------------------ | -------------------- | -------------------------------------------------------- |
+| `NativeSelect`           | `aria-hidden="true"` | 要素をアクセシビリティツリーから除外します。             |
+| `select` <Tag>内部</Tag> | `aria-readonly`      | `isReadOnly`が設定されている場合は`"true"`を設定します。 |
```
Comment (108yen) [inline docs/contents/components/forms/native-select/accessibility.ja.mdx:29]: - Please add role.
- Where is `aria-hidden`?

image

**Review by 108yen — CHANGES_REQUESTED** [review]
You should write a document to add `aria-label` to `NativeSelect`.

File: docs/contents/components/forms/native-select/accessibility.ja.mdx (hunk 1 above)
Comment (mei28) [reply docs/contents/components/forms/native-select/accessibility.ja.mdx:29]: I assumed aria-hidden=true would work based on the following code. Is this incorrect?" https://github.com/yamada-ui/yamada-ui/blob/46c97e371adc766101b2a1ef5d0c83270889f3ad/packages/components/native-select/src/native-select.tsx#L241

File: docs/contents/components/forms/native-select/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+tab: アクセシビリティ
+---
+
+`NativeSelect` は、アクセシビリティに関して、[WAI-ARIA - Combobox Pattern]("https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/")に従います。
+
+## キーボード操作
+
+| キー             | 説明                                                                                                       | 状態 |
+| ---------------- | ---------------------------------------------------------------------------------------------------------- | ---- |
+| `Tab`            | フォーカスがセレクトボックスに移動すると、最初の項目をフォーカスします。                                   | -    |
```
[the remark this answers is not in the document]
Comment (mei28) [reply docs/contents/components/forms/native-select/accessibility.ja.mdx:17]: I followed the documentation to write it, and I removed the description about tab." https://github.com/yamada-ui/yamada-ui/pull/3200

File: docs/contents/components/forms/native-select/accessibility.ja.mdx (hunk 1 above)
Comment (108yen) [reply docs/contents/components/forms/native-select/accessibility.ja.mdx:28]: This is `NativeSelectIcon`.
https://github.com/yamada-ui/yamada-ui/blob/46c97e371adc766101b2a1ef5d0c83270889f3ad/packages/components/native-select/src/native-select.tsx#L214-L251

**Comment by 108yen** [comment]
> You should write a document to add `aria-label` to `NativeSelect`.

Dont forget this too.

**Comment by mei28** [comment]
> > You should write a document to add `aria-label` to `NativeSelect`.
> 
> Dont forget this too.

I added aria-label="Native Select" because it was displayed that way in the inspect tool. Is it possible to confirm this label from the actual source code?

image

**Comment by 108yen** [comment]
@mei28 

Perhaps you misunderstand.
You should explain it like this.
https://yamada-ui.com/ja/components/forms/icon-button/accessibility

**Pull request closed without merging**